### PR TITLE
Doc 10395 Update Data Modeling Page with Scopes Collections

### PR DESCRIPTION
**Pull request by dinaabennett**
Added Scopes and Collections, link to Channels page, and updated the properties restrictions

### CODE REVIEW

File: modules/ROOT/pages/data-modeling.adoc [hunk 1]
```diff
@@ -67,13 +77,18 @@ You should change any top-level user properties that have a key with a leading u
 * Wrapping them inside another object with a key that doesn't have a leading underscore.
 
 
+
+
+
+
```
Comment (osfameron) [inline modules/ROOT/pages/data-modeling.adoc:83]: Lots of whitespace 😄 
(not a problem, it does help scan an asciidoc file, just made me wonder if there's anything about this in the style guide)

File: modules/ROOT/pages/data-modeling.adoc [hunk 2]
```diff
@@ -90,31 +105,51 @@ The document value is either:
 
 * A JSON value, termed a _Document_.
 +
-This JSON object is itself collection of key/value pairs, where the values may be numbers, strings, arrays or even nested objects themselves.
-As a result documents can represent highly-complex data structures in a readily parsable and self-organizing manner.
+This JSON object is a collection of key/value pairs. The values may be numbers, strings, arrays, or even nested objects.
```
Comment (osfameron) [inline modules/ROOT/pages/data-modeling.adoc:108]: New line for new sentence.

File: modules/ROOT/pages/data-modeling.adoc [hunk 3]
```diff
@@ -90,31 +105,51 @@ The document value is either:
...
-Couchbase Lite supports attachments of unlimited size, although the Sync Gateway currently imposes a 20MB limit for attachments synced to it.
+Couchbase Lite supports attachments of unlimited size, although the Sync Gateway imposes a 20MB limit for attachments synced to it.
 
 
 == Document Attributes
 Each _Document_ has the following attributes:
 
 * A document ID
-* A current revision ID (which changes every time the document is updated)
+* A current revision ID (which changes when the document is updated)
```
Comment (osfameron) [inline modules/ROOT/pages/data-modeling.adoc:120]: Hmm, that's interesting, I was worried that "when" might not be as precise as "every time", but I guess it is fine.
Does that "every time" *unnecessarily precise*? (And does that make it the opposite of a weasel word?)

File: modules/ROOT/pages/data-modeling.adoc [hunk 4]
```diff
@@ -90,31 +105,51 @@ The document value is either:
...
+
+== Scopes and Collections
+
+Scopes and collections allow you to organize your documents within a database.
+
+You can organize your documents in collections to logically group categories of content.
+
+You can organize your collections in scopes according to content-type or deployment-phase.
+
+xref:server:learn:data/scopes-and-collections.adoc[Couchbase Scopes and Collections for Server]
```
Comment (osfameron) [inline modules/ROOT/pages/data-modeling.adoc:142]: Should we introduce these links?

File: modules/ROOT/pages/data-modeling.adoc [hunk 5]
```diff
@@ -90,31 +105,51 @@ The document value is either:
...
+
+xref:couchbase-lite:java:scopes-collections-manage.adoc[Couchbase Scopes and Collections for Couchbase Lite]
+
+
+
+== Channels
+
+Sync Gateway uses xref:channels.adoc[Channels] to share a database’s documents and retain security access control.
```
Comment (osfameron) [inline modules/ROOT/pages/data-modeling.adoc:150]: Is this just here as a driveby IA improvement "we should add something about Channels to the data-modeling doc"?

Otherwise if this was added specifically for Scopes and Collections, then might be worth expanding a little? (e.g. whether Channels are at keyspace/collection level rather than per database etc.)

**Review by osfameron — APPROVED** [review]
👍 (minor comments inline)

**Review by franrob-projects — APPROVED** [review]
LGTM 👍
